content/2015-02-02-this-week-in-rust.md

```diff
@@ -233,4 +233,4 @@ Now you can follow breaking changes *[as they happen][BitRust]*!
 
 [n]: http://www.meetup.com/N-Languages-in-N-Months-NYC/events/218757056/
 [syd]: http://www.meetup.com/Rust-Sydney/events/220100853/
-[seattle]: https://www.eventbrite.com/e/mozilla-rust-seattle-meetup-tickets-12222326307?aff=erelexporg
\ No newline at end of file
+[seattle]: https://www.eventbrite.com/e/mozilla-rust-seattle-meetup-tickets-12222326307?aff=erelexporg
```

Comment:
Yeah, my editor added a line feed for the last line. I noticed this a bit too late.